config/config.go

```diff
@@ -173,6 +196,14 @@ func (c ServicePrincipalCredentials) Auth(opts azcore.ClientOptions) (azcore.Tok
 	return cred, nil
 }
 
+type WorkloadIdentityCredentials struct {
+	TenantID           string `toml:"tenant_id"`
+	ClientID           string `toml:"client_id"`
+	FederatedTokenFile string `toml:"federated_token_file"`
+	// AuthorityHost is not handled yet.
```

Comment:
If we don't need this now, we can just remove the commented code and add it later.